uqcsbot/scripts/crisis.py

```diff
@@ -0,0 +1,18 @@
+from uqcsbot import bot, Command
+
+
+@bot.on_command("crisis")
+@bot.on_command("mentalhealth")
+def handle_crisis(command: Command):
+    '''
+    `!crisis` or `!mentalhealth` - Get a list of mental health crisis resources.
+    '''
+    response = """*Mental health/crisis resources*
```

Comment:
For nicer indentation:

```suggestion
response = "\n".join(["*Mental health/crisis resources*",
		      "24/7 UQ Counselling and Crisis Line: 1300 851 998",
		      "Campus Security (emergency): 07 3365 3333",
		      "Campus Security (non-emergency): 07 3365 1234",
		      "Counselling Services: https://www.uq.edu.au/student-services/counselling-services",
		      "UQ Psychology Clinic: https://clinic.psychology.uq.edu.au/therapies-and-services",	    
		      "UQ Resources: https://about.uq.edu.au/campaigns-and-initiatives/mental-health"])
```

Reply:
Alternatively move the string outside the body of the function for nice indenting and nice text formatting

Reply:
I like Tom's a bit more. `response`, `RESPONSE` or?